CustomDictionary.xml

```diff
@@ -71,6 +71,7 @@
       <Word>SingletonAttribute</Word>
       <Word>Prefetch</Word>
       <Word>ScopeId</Word>
+      <Word>Poco</Word>
```

Comment:
nit: I still see some warnings in IConverterManager and ITriggerBindingStrategy for the word POCO. I think you need to add Poco to the Settings.StyleCop word list in the repo root. Just add there and fix the rest of the warnings please.